### ISSUE TITLE
Strange behavior of the time filter. Shows changes outside of the specified time period.

### ISSUE THREAD

**Issue by alexander-ii** [tyrasd/latest-changes#6]
When select `last week` option, in the list you can see changes that were made earlier than just last week.

https://tyrasd.github.io/latest-changes/#19/55.76096/37.63826

![image](https://user-images.githubusercontent.com/25887642/36517704-40059a58-1794-11e8-99e5-f8fe85bb56c3.png)

**Comment by tyrasd** [comment]
yeah, I've noticed this a few times as well. Normally it goes away after panning around a bit. Have to investigate why it happens…

**Comment by ximex** [comment]
i also had this behaviour
